app.rb

```diff
@@ -0,0 +1,116 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
```

Comment:
- [ ] Good job so far 👍👍 However, when I'm trying to run the app, the following error is occurring ⚠ and I am unable to see if the app is behaving as [expected](https://www.youtube.com/watch?v=vkkgrhD6aXQ). Kindly fix this 🤓⛏

_Hint: You do not have any files named `book.rb` and `rental.rb`_

![image](https://github.com/Javed-Shahbaz/school_library/assets/106119105/f3b33f99-f4ea-4000-9bb6-f52970968070)